src/azure-cli-core/azure/cli/core/commands/__init__.py

```diff
@@ -415,6 +438,13 @@ def get_command_table(module_name=None):
         logger.debug("Loaded all modules in %.3f seconds. "
                      "(note: there's always an overhead with the first module loaded)",
                      cumulative_elapsed_time)
+    try:
+        # We always load extensions even if the appropriate module has been loaded
+        # as an extension could override the commands already loaded.
```

Comment:
The premise that extension could act as a temporary upgrade to a stable version of the commands is justified. However, I would prefer we have some kind of UI presentation to remind user if a command is replaced by extension.

Side question, what will happen if two extensions' commands names collide?